holoviews/plotting/renderer.py

```diff
@@ -90,34 +92,42 @@ class Renderer(Exporter):
         The full, lowercase name of the rendering backend or third
         part plotting package used e.g 'matplotlib' or 'cairo'.""")
 
-    dpi=param.Integer(None, doc="""
+    dpi = param.Integer(None, doc="""
         The render resolution in dpi (dots per inch)""")
 
     fig = param.ObjectSelector(default='auto', objects=['auto'], doc="""
         Output render format for static figures. If None, no figure
         rendering will occur. """)
 
-    fps=param.Number(20, doc="""
+    fps = param.Number(20, doc="""
         Rendered fps (frames per second) for animated formats.""")
 
     holomap = param.ObjectSelector(default='auto',
                                    objects=['scrubber','widgets', None, 'auto'], doc="""
         Output render multi-frame (typically animated) format. If
         None, no multi-frame rendering will occur.""")
 
-    mode = param.ObjectSelector(default='default', objects=['default'], doc="""
-         The available rendering modes. As a minimum, the 'default'
-         mode must be supported.""")
+    mode = param.ObjectSelector(default='default',
+                                objects=['default', 'server'], doc="""
+        Whether to render the object in regular or server mode. In server
+        mode a bokeh Document will be returned which can be served as a
+        bokeh server app. By default renders all output is rendered to HTML.""")
 
-    size=param.Integer(100, doc="""
+    size = param.Integer(100, doc="""
         The rendered size as a percentage size""")
 
+    widget_location = param.ObjectSelector(default=None, allow_None=True, objects=[
```

Comment:
It would be nice to have a shorted name than `widget_location` but I don't currently have any better suggestions...